test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -431,6 +434,30 @@ Describe "Additional tests for Import-Module with WinCompat" -Tag "CI" {
             pwsh -NoProfile -NonInteractive -c "[System.Management.Automation.Internal.InternalTestHooks]::SetTestHook('TestWindowsPowerShellPSHomeLocation', `'$basePath`');Import-Module $ModuleName -WarningAction Ignore" *> $LogPath
             $LogPath | Should -Not -FileContentMatch 'loaded in Windows PowerShell'
         }
+
+        It "Fails to import incompatible module if implicit WinCompat is disabled in config " -Skip:(-not $IsWindows) {
```

Comment:
Rather than skipping individual tests, all of these tests are Windows only, so just set the default params in the BeforeAll and restore them in AfterAll

Reply:
Good point. Also, after sync with @adityapatwardhan I updated one more instance in this test file where skipping logic was violating guidelines.
Also updated these tests to be 'Feature' tests so that it is more in-line with priorities of other similar tests in this test file.